guides/common/modules/con_use-case-considerations-for-foreman-on-aws.adoc

```diff
@@ -0,0 +1,5 @@
+[id="use-case-considerations-for-{project-context}-on-aws"]
+= Use case considerations for {Project} on AWS
+
+Amazon Web Services (AWS) is an image-only compute resource which means that there are common {Project} use cases that do not work or require extra configuration in your Amazon Web Service environment.
```

Comment:
Is this really relevant? You can simply take a bare OS image and then manage it as a normal OS in EC2.